driver/src/test/java/org/neo4j/driver/ConfigTest.java

```diff
@@ -370,7 +400,7 @@ void shouldErrorWithInvalidUserAgent()
         void shouldSerialize() throws Exception
         {
             Config config = Config.builder()
-                    .withMaxConnectionPoolSize( 123 )
+                                  .withMaxConnectionPoolSize( 123 )
```

Comment:
```suggestion
                    .withMaxConnectionPoolSize( 123 )
```

Reply:
Updated formatting in https://github.com/neo4j/neo4j-java-driver/pull/1137/commits/d942e0deb9a5f26a6dda13fe2258723aea80980d.